File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.swift
class Solution {
    func findMedianSortedArrays(_ nums1: [Int], _ nums2: [Int]) -> Double {
        let a = nums1 + nums2
        let b = a.sorted()
        let len = b.count

        if len % 2 != 0 {
            return Double(b[len / 2])
        } else {
            return Double((b[len / 2] + b[len / 2 - 1])) / Double(2)
        }
    }
}

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        newArr = nums1 + nums2
        newArr.sort()
        
        if len(newArr) == 1:
            return newArr[0]
        length = len(newArr)

        return newArr[length//2] if length%2 == 1 else (newArr[length//2-1] + newArr[length//2])/2